<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>913</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Résérvations</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>891</width>
      <height>661</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>gestion résérvations</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>70</y>
        <width>581</width>
        <height>431</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>ajouter</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>20</y>
          <width>231</width>
          <height>181</height>
         </rect>
        </property>
        <property name="title">
         <string>Ajouter</string>
        </property>
        <widget class="QPushButton" name="pushButton_ajouter">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>90</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton::hover
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 207, 179, 255),stop:1 rgb(108, 135, 255));

}</string>
         </property>
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
        <widget class="QWidget" name="">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>118</width>
           <height>134</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>num</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="num"/>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>heure</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="heure"/>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>nbrp</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="nbrp"/>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>afficher</string>
       </attribute>
       <widget class="QTableView" name="tableView_aff">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>431</width>
          <height>271</height>
         </rect>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>290</y>
          <width>501</width>
          <height>141</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="title">
         <string>Envoyer un mail</string>
        </property>
        <widget class="QLineEdit" name="le_envoyer">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>40</y>
           <width>131</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
        <widget class="QPushButton" name="envoyer">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>70</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton::hover
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 207, 179, 255),stop:1 rgb(108, 135, 255));

}</string>
         </property>
         <property name="text">
          <string>Envoyer</string>
         </property>
        </widget>
        <widget class="QTextEdit" name="te_mail">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>70</y>
           <width>241</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_objet">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>20</y>
           <width>241</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
        <widget class="QLabel" name="label_47">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>30</y>
           <width>47</width>
           <height>14</height>
          </rect>
         </property>
         <property name="text">
          <string>Objet</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_48">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>21</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>À</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_15">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>170</y>
          <width>91</width>
          <height>131</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="title">
         <string>TRI</string>
        </property>
        <widget class="QWidget" name="">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>77</width>
           <height>83</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="Tri_Poid">
            <property name="styleSheet">
             <string notr="true">QPushButton::hover
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 207, 179, 255),stop:1 rgb(108, 135, 255));

}</string>
            </property>
            <property name="text">
             <string>Nbrp</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Tri_Taille">
            <property name="styleSheet">
             <string notr="true">QPushButton::hover
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 207, 179, 255),stop:1 rgb(108, 135, 255));

}</string>
            </property>
            <property name="text">
             <string>Heure</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Tri_IdE">
            <property name="styleSheet">
             <string notr="true">QPushButton::hover
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 207, 179, 255),stop:1 rgb(108, 135, 255));

}</string>
            </property>
            <property name="text">
             <string>Num</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QLineEdit" name="Psearch_5">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>50</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_81">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>30</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Rechercher:</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="widget">
       <attribute name="title">
        <string>Suprimer</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>501</width>
          <height>331</height>
         </rect>
        </property>
        <property name="title">
         <string>Suprime</string>
        </property>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>50</y>
           <width>47</width>
           <height>14</height>
          </rect>
         </property>
         <property name="text">
          <string>num:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_idS">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>50</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_S">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>120</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton::hover
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 207, 179, 255),stop:1 rgb(108, 135, 255));

}</string>
         </property>
         <property name="text">
          <string>Suprime</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>modifier</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_modif">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>70</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton::hover
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 207, 179, 255),stop:1 rgb(108, 135, 255));

}</string>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="numnv">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>70</y>
          <width>201</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>70</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Num</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="nbrp_3">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>200</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="heure_3">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>110</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_23">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>110</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>heure</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_24">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>200</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>nbrp</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="label_ph">
      <property name="geometry">
       <rect>
        <x>-220</x>
        <y>-120</y>
        <width>1351</width>
        <height>761</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>label_ph</zorder>
     <zorder>tabWidget_2</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>gestion commandes</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>461</width>
        <height>401</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>ajouter</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>30</y>
          <width>461</width>
          <height>361</height>
         </rect>
        </property>
        <property name="title">
         <string>Ajouter</string>
        </property>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>110</y>
           <width>47</width>
           <height>14</height>
          </rect>
         </property>
         <property name="text">
          <string>id</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>180</y>
           <width>47</width>
           <height>14</height>
          </rect>
         </property>
         <property name="text">
          <string>adresse</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>240</y>
           <width>47</width>
           <height>14</height>
          </rect>
         </property>
         <property name="text">
          <string>plat</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="id">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>110</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="adresse">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>180</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="plat">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>239</y>
           <width>113</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_ajouter_2">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>180</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton::hover
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 207, 179, 255),stop:1 rgb(108, 135, 255));

}</string>
         </property>
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>290</y>
           <width>47</width>
           <height>14</height>
          </rect>
         </property>
         <property name="text">
          <string>prix</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="prix">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>289</y>
           <width>113</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>afficher</string>
       </attribute>
       <widget class="QTableView" name="tableView_aff_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>321</width>
          <height>321</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="widget_2">
       <attribute name="title">
        <string>Suprimer</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>501</width>
          <height>331</height>
         </rect>
        </property>
        <property name="title">
         <string>Suprime</string>
        </property>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>50</y>
           <width>47</width>
           <height>14</height>
          </rect>
         </property>
         <property name="text">
          <string>id:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_idS_2">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>50</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_S_2">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>120</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton::hover
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 207, 179, 255),stop:1 rgb(108, 135, 255));

}</string>
         </property>
         <property name="text">
          <string>Suprime</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>modifier</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_modif_2">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>70</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton::hover
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 207, 179, 255),stop:1 rgb(108, 135, 255));

}</string>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="numnv_2">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>70</y>
          <width>201</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>70</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>id</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="label_ph_2">
      <property name="geometry">
       <rect>
        <x>-100</x>
        <y>-110</y>
        <width>1071</width>
        <height>751</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>label_ph_2</zorder>
     <zorder>tabWidget_3</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>913</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
